spring-cloud-gcp-pubsub-stream-binder/src/main/java/org/springframework/cloud/gcp/stream/binder/pubsub/provisioning/PubSubProducerDestination.java

```diff
@@ -1,17 +1,17 @@
 /*
- *  Copyright 2017 original author or authors.
+ * Copyright 2017-2017 the original author or authors.
```

Comment:
There many of these single year ranges. Is that correct?

Reply:
Yes, but see Artem's comments. We're changing it to 2017-2018.
Are you requesting changes to the PR?

Reply:
Would just "2017" be acceptable?

Reply:
Not according to Spring Boot header style.

Reply:
No, see related discussion in the issue for this PR.